Controllers/CustomerController.cs

```diff
@@ -46,8 +45,7 @@ public string Details(int CustomerId)
                     // var objList2 = (IEnumerable<Customer>)_db.Customer.Where(b=>b.CustomerId==CustomerId);
```

Comment:
var obj = _db.Customer.Find(CustomerId);

//make this statement before if statement ,then you can use it , instead of fetching two times you can store once and use.

Reply:
Yes looks great, will do. Thanks!